trap.go

```diff
@@ -188,6 +188,26 @@ func (t *TrapListener) Listen(addr string) error {
 			if traps != nil {
 				t.OnNewTrap(traps, remote)
 			}
+			if traps.PDUType == InformRequest {
+				traps.PDUType = GetResponse
+				log.Printf("sending INFORM response: %+v\n", traps)
```

Comment:
Okay for debugging but too much logging for the final change.

Reply:
I think I'd meant to strip this out before finishing, will make sure to do so.